src/main/webapp/script.js

```diff
@@ -367,17 +371,67 @@ function getFailedJobsCost(aggregated) {
         failedCost += aggregated[category][costs].price;
       }
     }
-    jobData.push(failedCost);
-    data.push(jobData);
-    isAllZero = isAllZero && ((failedCost == 0) || isNaN(failedCost));
```

Comment:
`isAllZero = isAllZero && ((failedCost === 0) || isNaN(failedCost));`

Reply:
Good point! This is a deleted line but I'll make sure to use this in the future :)